documentation/en/default-lotus-config.toml

```diff
@@ -342,3 +342,11 @@
   #ActorEventDatabasePath = ""
 
 
+[EthTxHashConfig]
```

Comment:
* We're in the config, calling things Config here is a bit redundant
* Group name ideally shouldn't be that narrow.
* I'd call this group `FEVM`/`EVM` or something broad like that given that we're passing this config into evm module.